Content.Client/Eye/Blinding/BlurryVisionOverlay.cs

```diff
@@ -0,0 +1,69 @@
+using Robust.Client.Graphics;
+using Robust.Client.Player;
+using Robust.Shared.Enums;
+using Robust.Shared.Prototypes;
+using Content.Shared.Eye.Blinding;
+
+namespace Content.Client.Eye.Blinding
+{
+    public sealed class BlurryVisionOverlay : Overlay
+    {
+        [Dependency] private readonly IPrototypeManager _prototypeManager = default!;
+        [Dependency] IEntityManager _entityManager = default!;
+        [Dependency] private readonly IPlayerManager _playerManager = default!;
+
+
+        public override bool RequestScreenTexture => true;
+        public override OverlaySpace Space => OverlaySpace.WorldSpace;
+        private readonly ShaderInstance _blurryVisionXShader;
+        private readonly ShaderInstance _blurryVisionYShader;
+        private BlurryVisionComponent _blurryVisionComponent = default!;
+
+        public BlurryVisionOverlay()
+        {
+            IoCManager.InjectDependencies(this);
+            _blurryVisionXShader = _prototypeManager.Index<ShaderPrototype>("BlurryVisionX").InstanceUnique();
+            _blurryVisionYShader = _prototypeManager.Index<ShaderPrototype>("BlurryVisionY").InstanceUnique();
```

Comment:
why do these have to be two different shaders?

Reply:
I'm not a shader guy I just copied from godot documentation

Reply:
All of the more obvious ways to do it do not work on our rendering engine (at least not for older machines)